pkgs/development/python-modules/dynaconf/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  python,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+  flask,
+  django,
+  iniSupport ? false,
+  configobj,
+  redisSupport ? false,
+  redis,
+  vaultSupport ? false,
+  hvac,
+}:
+buildPythonPackage rec {
+  pname = "dynaconf";
+  version = "3.1.11";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.8";
+  doCheck = false;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-2c+1D9SnGlQ/0jhF1PWFtiC2/22dPMGCXGFPeyCXyzk=";
+  };
+
+  propagatedBuildInputs =
+    [setuptools]
```

Comment:
I think this is needed since without it this stacktrace appeared when attempting to import this package.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nix/store/w27jgd5gnsvf03kfk67y19sc9vrk4npa-python3.11-dynaconf-3.1.11/lib/python3.11/site-packages/dynaconf/__init__.py", line 5, in <module>
    from dynaconf.contrib import DjangoDynaconf  # noqa
  File "/nix/store/w27jgd5gnsvf03kfk67y19sc9vrk4npa-python3.11-dynaconf-3.1.11/lib/python3.11/site-packages/dynaconf/contrib/__init__.py", line 4, in <module>
    from dynaconf.contrib.flask_dynaconf import DynaconfConfig  # noqa
  File "/nix/store/w27jgd5gnsvf03kfk67y19sc9vrk4npa-python3.11-dynaconf-3.1.11/lib/python3.11/site-packages/dynaconf/contrib/flask_dynaconf.py", line 17, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

I'm unsure why `pythonImportsCheck` didn't catch this, but I did see it in a nix-shell I instantiated with this commit.